src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -810,8 +825,9 @@ void goto_analyzer_parse_optionst::help()
     "\n"
     "Abstract interpreter options:\n"
     // NOLINTNEXTLINE(whitespace/line_length)
-    " --location-sensitive         use location-sensitive abstract interpreter\n"
-    " --concurrent                 use concurrency-aware abstract interpreter\n"
+    " --location-sensitive         one domain per location\n"
+    " --location-insensitive       one domain per function\n"
+    " --concurrent                 one domain per location and concurrency analysis\n"
```

Comment:
Is local-insensitive / concurrency-aware a  sensible combo?

Reply:
Yes but at the moment none of the domains that work with ```--concurrent``` are integrated into goto-analyzer so it's more of a TODO than a pressing issue.